package.json

```diff
@@ -49,8 +51,8 @@
     "flow-bin": "^0.63.1",
     "glob": "^7.1.2",
     "husky": "^0.14.3",
-    "jest": "^22.4.3",
-    "jest-canvas-mock": "^1.0.2",
+    "jest": "^23.6.0",
```

Comment:
Jest needs to be upgraded to upgrade Babel?

Reply:
No, I noticed jest was still installing the babel 6 dependencies so I upgraded, but it still was after the upgrade.

Reverted, but had to change a config var `testURL` to `'http://localhost/' because it was giving an error without this (ps: it's the default on newer jests).